.clj-kondo/config.edn

```diff
@@ -33,9 +32,9 @@
   ;;
   ;; disabled linters
   ;;
-
-  :unexpected-recur         {:level :off} ; TODO (cam): I think we just need to tell it how to handle MBQL match and we can enable this?
-  :used-underscored-binding {:level :off} ; false positives: see https://github.com/clj-kondo/clj-kondo/issues/2152
+  :non-arg-vec-return-type-hint {:level :off} ; Moving the type hints to the arg lists makes clj-kondo happy, but breaks eastwood (and maybe causes reflection warnings) at the call sites.
```

Comment:
Instead of disabling this linter in Kondo let's just disable the Eastwood linter. We don't need both Kondo and Eastwood to lint it and Kondo is preferrable to Eastwood. Type hints are 100% supposed to go on the arg lists, not sure what you mean about them maybe causing reflection warnings or making Eastwood upset, we're doing them in lots of other places and they work fine everywhere else

Reply:
I blindly trusted Braden's comment [here](https://github.com/metabase/metabase/pull/44456/files#diff-753579040dda77301062d09ffbe122bf5d6dd30b35d412760d8f1d64e0fe8e0cL434-L436) on this, but he might have just been putting the type hints in the wrong place.

I've fixed them here 5dfe03915caae54382645814d4802686cc2df4a5